doc/project/checklist.md

```diff
@@ -262,6 +262,10 @@ Set up FPV regression by adding the module to `hw/formal/fpv_all` script.
 
 Sub-modules that are pre-verified with their own testbenches have already reached V1 or higher stage.
 
+### TB_LINT_PASSING
+
+VeribleLint for the testbench setup, runs, and passes with common waiver.
```

Comment:
```suggestion
[VeribleLint](add-link-here) for the testbench is [setup](add-link-to-opentitan-linting-doc-if-applicable) to run in nightly regressions, with appropriate waivers.
```

Also, add path to the dv lint hjson cfg where an entry for this is expected to be added (make it generic, like `hw/<top-level-design>/lint/<top-level-design>_dv_lint_cfgs.hjson`.

Reply:
Thanks for the suggestion, I will update them as you recommended.